packages/ProcurementML/src/ValueObjects/VendorRecommendationEligibleCandidate.php

```
@@ -0,0 +1,97 @@
+        public array $deterministicReasons,
+        public array $llmInsights = [],
+        public array $warningFlags = [],
+        public array $warnings = [],
+    ) {
+        $this->assertNonEmpty($this->vendorId, 'eligible vendor id');
+        $this->assertNonEmpty($this->vendorName, 'eligible vendor name');
+        $this->assertNonEmpty($this->confidenceBand, 'eligible confidence band');
+        $this->assertNonEmpty($this->providerExplanation, 'eligible provider explanation');
+
+        if ($this->fitScore < 0 || $this->fitScore > 100) {
+            throw ProcurementMlContractException::invalidValue('eligible candidate fit score');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Nexus\ProcurementML\Exceptions\ProcurementMlContractException' in method '__construct'. (undefined)

(StaticAccess)